neuro_flow/storage.py

```diff
@@ -295,8 +311,15 @@ async def create_bake(
         )
         bake_uri = _mk_bake_uri(bake)
         await self._client.storage.mkdir(bake_uri, parents=True)
-        config_uri = bake_uri / config_name
-        await self._write_file(config_uri, config_content)
+
+        # Upload all configs
+        configs_dir = bake_uri / "configs"
+        await self._client.storage.mkdir(configs_dir, parents=True)
+        for config_path, config_content in configs:
+            await self._client.storage.mkdir(
+                configs_dir / str(config_path.parent), parents=True
+            )
+            await self._write_file(configs_dir / str(config_path), config_content)
 
         await self._write_json(bake_uri / "00.init.json", _bake_to_json(bake))
```

Comment:
Save paths to all stored configs in `bake` object, write them to `00.init.json` file.
It allows eliminating the directory lookup on config fetching.